pootle/templates/browser/index.html

```diff
@@ -143,10 +143,16 @@ <h3 class="top">{% trans "Updates" %}</h3>
 {% endif %}
 
 <div id="autorefresh-notice">
-  {% blocktrans trimmed %}
-  Some data on this page is currently being calculated, and the page will be
-  refreshed automatically in <strong>30 seconds</strong>.
-  {% endblocktrans %}
+  {% if refresh_dirty_stats %}
+    {% blocktrans trimmed %}
+    Some data on this page is currently being calculated, and the page will be
+    refreshed automatically in <strong>30 seconds</strong>.
+    {% endblocktrans %}
+  {% else %}
+    {% blocktrans trimmed %}
+    Some data on this page is currently being calculated.
+    {% endblocktrans %}
```

Comment:
I thought the idea was not to show this at all. But if that is the case you can fit the whole translatable string in a single line:

```
    {% trans "Some data on this page is currently being calculated." %}
```